### PR TITLE
Add compat data for CanvasCaptureMediaStream

### PR DESCRIPTION
**Pull request by maboa**
First PR here. Hope I'm doing it right :)

### CODE REVIEW

**Comment by teoli2003** [comment]
Welcome on board Mark! Travis-ci indicates that you urls doesn't match the pattern (we don't put /en-US/ in them).

**Comment by maboa** [comment]
Thanks Jean-Yves - I have now discovered the linter! :) Also added some notes.

File: api/CanvasCaptureMediaStream.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "41",
+            "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: We don't use note for indicating disabled by default feature. See https://github.com/mdn/browser-compat-data/blob/master/compat-data-schema.md#flag for using the "flag" object.

File: api/CanvasCaptureMediaStream.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+          },
+          "firefox_android": {
+            "version_added": "41",
+            "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+          },
+          "ie": {
+            "version_added": false
+          },
+          "ie_mobile": {
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: We will remove "ie_mobile" in a couple of hours, please remove this one.

File: api/CanvasCaptureMediaStream.json [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "41",
+            "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+          },
+          "firefox_android": {
+            "version_added": "41",
+            "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: Use "flag"

File: api/CanvasCaptureMediaStream.json [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: Use "flag"

File: api/CanvasCaptureMediaStream.json [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+            },
+            "firefox_android": {
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: Use "flag"

File: api/CanvasCaptureMediaStream.json [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: Use "flag".

File: api/CanvasCaptureMediaStream.json [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+            },
+            "firefox_android": {
+              "version_added": "34"
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: This looks dodgy. I think this was a bad cut and paste, put "41" and the flag there, too.

File: api/CanvasCaptureMediaStream.json [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+            },
+            "firefox_android": {
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: Remove "ie_mobile"

File: api/CanvasCaptureMediaStream.json [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+              "version_added": "41",
+              "notes": "Disabled by default; set the preference <code>canvas.capturestream.enabled</code> to <code>true</code> to activate."
+            },
+            "firefox_android": {
+              "version_added": "34"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
```
Comment (teoli2003) [inline api/CanvasCaptureMediaStream.json]: Remove "ie_mobile"

**Comment by maboa** [comment]
Note I switched `CanvasCaptureMediaStream/requestFrame`

           "firefox_android": {
              "version_added": null
            },
As it didn't make sense to me, that support was added before `CanvasCaptureMediaStream`  https://developer.mozilla.org/en-US/docs/Web/API/CanvasCaptureMediaStream/requestFrame

**Pull request merged**